libbeat/docs/command-reference.asciidoc

```diff
@@ -15,30 +15,32 @@
 :global-flags: Also see <<global-flags,Global flags>>.
 
 :deploy-command-short-desc: Deploys the specified function to your serverless environment
+
+ifndef::no_dashboards_apm[]
 :export-command-short-desc: Exports the configuration, index template, or a dashboard to stdout
+endif::no_dashboards_apm[]
+
+ifdef::no_dashboards_apm[]
```

Comment:
Instead of introducing a new attribute called `no_dashboards_apm` why not use the `no_dashboards` attribute? Unless I'm overlooking something, I think all the statements you wrap in `no_dashboards_apm` are also valid for Beats. It makes sense for Beats to leverage the extra work you've done here. :-)

Reply:
Ahh, perfect, I should've asked. I'll make that change!